components/Carousel/Carousel.js

```diff
@@ -1,14 +1,44 @@
 class Carousel {
-
+    constructor(el) {
+        this.el = el;
+        this.rightButton = document.querySelector('.right-button');
+        this.leftButton = document.querySelector('.left-button');
+        this.count = 0
+        this.rightButton.addEventListener('click', () => this.rightClick());
+        this.leftButton.addEventListener('click', () => this.leftClick());
+    }
+    start() {
+        const images = document.querySelectorAll('.carousel img');
+        images[0].classList.add('show');
+    }
+    rightClick() {
+        const images = document.querySelectorAll('.carousel img');
+        images.forEach((img) => {
+            img.classList.remove('show');
+        })
+        if (this.count < images.length - 1) {
+            this.count += 1;
+            images[this.count].classList.add('show');
+        } else {
+            this.count = 0;
+            images[this.count].classList.add('show');
+        }
+    }
+    leftClick() {
+        const images = document.querySelectorAll('.carousel img');
+        images.forEach((img) => {
+            img.classList.remove('show');
+        })
+        if (this.count > 0) {
+            this.count -= 1;
+            images[this.count].classList.add('show');
+        } else {
+            this.count = images.length - 1;
+            images[this.count].classList.add('show');
+        }
+    }
 }
 
-let carousel = document.querySelector();
-
-/* If You've gotten this far, you're on your own! Although we will give you some hints:
-    1. You will need to grab a reference to the carousel, and in it grab the laft and right buttons
-    2. You will need to grab a reference to all of the images
-    3. Create a current index
-    4. Those buttons are gonna need some click handlers.
-    5. Think of how you would animate this compoennt. Make the cards slide in and out, or fade. It's up to you!
-    6. Have fun!
-*/
\ No newline at end of file
+let carousel = document.querySelector('.carousel');
+carousel = new Carousel(carousel);
+carousel.start()
```

Comment:
I really like that instead of just calling the first item in the array you chose to use a method!